Helewud/zc_core
# Update a member profile

### **What does this PR do?**

- Allows the member of an organization to update their profile.

### **Task completed**

- Update profile: `PATCH '/organizations/{id}/members/{mem_id}/profile'`

### **How should this be tested?**

With postman, perform the following tasks:

- Update user: `PATCH '/organizations/{id}/members/{mem_id}/profile'` with a organization id  `id` and member id `mem_id` and a valid post body(check profile struct for accepted fields). Try this endpoint with badly structured or invalid JSON data too.
